src/styles/index.scss

```diff
@@ -1,3 +1,131 @@
+$base-color: #060b35;
+$secondary-color: #616070;
+$button-buy-color: #00acdc;
+$button-buy-activecolor: #fff;
+
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
```

Comment:
[STYLES] - Remember to use fallback fonts - alternative font-family in case the main one doesn't work [like this](https://www.w3schools.com/cssref/pr_font_font-family.asp)
![image](https://github.com/mate-academy/layout_product-cards/assets/63243136/374a498a-254b-4e0c-859e-9cc22aa54c76)